contentstack/build.gradle

```diff
@@ -4,12 +4,13 @@ plugins {
 }
 
 import com.vanniktech.maven.publish.SonatypeHost
+
 android.buildFeatures.buildConfig true
 
 mavenPublishing {
     publishToMavenCentral(SonatypeHost.DEFAULT)
     signAllPublications()
-    coordinates("com.contentstack.sdk", "android", "3.14.0")
+    coordinates("com.contentstack.sdk", "android", "3.13.0")
```

Comment:
@harshithad0703 This should be 3.15.0 right? if it's release version.

Reply:
in mvn repo 3.13.0 is the latest, will check and update